src/lib/autocomplete/BUILD.bazel

```diff
@@ -0,0 +1,40 @@
+package(default_visibility=["//visibility:public"])
+load("@angular//:index.bzl", "ng_module")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_library")
+load("@io_bazel_rules_sass//sass:sass.bzl", "sass_binary")
```

Comment:
you can load these two symbols in one load statement

Reply:
Done